sdk/src/system_accounts.rs

```diff
@@ -0,0 +1,143 @@
+use std::marker::PhantomData;
+
+use solana_program::{account_info::AccountInfo, instruction::AccountMeta};
+
+#[repr(usize)]
+pub enum LightSystemAccountIndex {
```

Comment:
You are probably aware just a reminder for final review, there is a duplicate enum in `instruction_accounts`.

Reply:
Yes, good point. I kept this one and removed the other one.